src/formatting.rs

```diff
@@ -99,8 +99,8 @@ fn format_project<T: FormatHandler>(
 
     let mut context = FormatContext::new(&krate, report, parse_session, config, handler);
     let files = modules::ModResolver::new(
-        context.parse_session.source_map(),
-        directory_ownership.unwrap_or(parse::DirectoryOwnership::UnownedViaMod(false)),
+        &context.parse_session,
+        directory_ownership.unwrap_or(parse::DirectoryOwnership::UnownedViaMod(true)),
```

Comment:
It is set to `true` to allow modules to be defined in places where result in an error otherwise.

https://github.com/rust-lang/rust/blob/7cdaffd7962c4aae0cadd82baa241901b03f9458/src/libsyntax/parse/parser.rs#L6654-L6658